test/docker_questdb.py

```
@@ -0,0 +1,195 @@
+
+    def _await_http_up(self):
+        def check_http_up():
+            if not self._container_running():
+                raise RuntimeError('QuestDB container exited during startup.')
+            req = urllib.request.Request(
+                f'http://{self.host}:{self.http_server_port}/ping',
+                method='GET')
+            try:
+                resp = urllib.request.urlopen(req, timeout=1)
+                return resp.status == 204
+            except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)